### PR TITLE
Add cart and wishlist feature

### PR DESCRIPTION
**Pull request by abhisheksahani-nova**
Implemented Cart , Wishlist , Login and Signup api integration .

### CODE REVIEW

File: src/context/wishlist-context.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+const WishlistContext = createContext();
+
+const WishlistProvider = ({ children }) => {
+  const [wishlistProducts, setWishlistProducts] = useState([]);
+
+  return (
+    <WishlistContext.Provider value={{ wishlistProducts, setWishlistProducts }}>
+      {children}
+    </WishlistContext.Provider>
```
Comment (anik31) [inline src/context/wishlist-context.js:11]: appreciation - liked that you're using separate context for cart & wishlist

**Review by anik31 — APPROVED** [review]
Everything looks good 👍 

suggestion - you can divide larger files into smaller ones

File: src/context/wishlist-context.js (hunk 1 above)
Comment (abhisheksahani-nova) [reply src/context/wishlist-context.js:11]: Thanks for taking time and reviewing this , I will work on the suggestion given .

**Review by khushijohri2001 — APPROVED** [review]
Code looks good to me👍

**Pull request merged**